client/components/promo-card/index.jsx

```diff
@@ -0,0 +1,21 @@
+import { Card, Button } from '@automattic/components';
+import FormattedHeader from 'calypso/components/formatted-header';
+
+import './style.scss';
+
+const PromoCard = ( { headerText, contentText, image, ctaText, onClick } ) => {
+	return (
+		<Card className="promo-card">
+			<img src={ image } alt="" aria-hidden="true" />
```

Comment:
@DustyReagan is the `aria-hidden` intentional?

Reply:
Hey Matias since it is a Decorative Image, I added it but after [reading this](https://www.w3.org/WAI/tutorials/images/decorative/), I think is enough with the alt prop being null. What do you think?

Reply:
Yeah, the empty alt value is better. You can also consider adding `role="presentation"` alongside it, but aria-hidden seems wrong for this case!

Reply:
Got it will fix it on a subsequent PR! thanks for the review @mtias

Reply:
PR is here :) https://github.com/Automattic/wp-calypso/pull/66272